Fixed doc reference to telemetry microservice

GSchroder opened the pull request:
Had no reference to repository and was referring to "telemetry-agent" instead of "telemetry"

azurecla commented:
Hi __@ravipal__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ravipal). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PRmerger commented:
@ravipal : Thanks for your contribution to the Azure documentation! The author, @seanmck, has been notified to review your proposed change.

MikkelHegn commented:
#sign-off @ravipal @tysonn

GSchroder committed:
Fixed doc reference to telemetry microservice

Had no reference to repository and was referring to "telemetry-agent" instead of "telemetry"
